digdag-core/src/main/java/io/digdag/core/workflow/TaskControl.java

```diff
@@ -121,7 +121,7 @@ private static long addTasks(TaskControlStore store,
 
         Map<String, ResumingTask> resumingTaskMap = resumingTasks
             .stream()
-            .collect(Collectors.toMap(t -> t.getFullName(), t -> t));
+            .collect(Collectors.toMap(t -> t.getFullName(), t -> t, (a, b) -> a));
```

Comment:
Looks good! May I ask you to add comment in the source code about why we need to add this fix. It would be helpful for future maintenance. It's fine to copy and paste from the description in your PR.